CONTRIBUTING.rst

```diff
@@ -239,24 +240,29 @@ Debian/Ubuntu
       buildscripts/scons.py -j$(nproc --all) --jlink=2 --install-mode=legacy --disable-warnings-as-errors --ssl --opt=on --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib" mongod
 
 This command builds only the database. Other available targets for the
-``scons`` command are: - ``mongod`` - ``mongos`` - ``mongo`` - ``core``
-(includes ``mongod``, ``mongos``, ``mongo``) - ``all``
+``scons`` command are:
+
+- ``install-mongod`` 
+- ``install-mongos`` 
+- ``install-mongo`` 
+- ``install-core`` (includes ``mongod``, ``mongos``, ``mongo``) 
+- ``install-all``
```

Comment:
For version 4.4 no need to add `install-` prefix when `scons.py` is executed with `--install-mode=legacy`